gcp_delete_old_images.py

```diff
@@ -57,12 +59,59 @@ def delete_old_images():
     subprocess.run(delete_cmd, check=True)
 
 
+def delete_old_docker_images_helper(delete_images, base_cmd, with_tags=False):
+    if not delete_images:
+        return
+
+    delete_images = [f'{image_name[0]}@{image_name[1]}' for
+                     image_name in delete_images]
+
+    print(f"deleting docker images {'with tags' if with_tags else 'without tags'}:\n" +
+          '\n'.join(delete_images))
+
+    for image in delete_images:
+        # '--delete-tags' flag shouldn't be added when deleting images without
+        # tags to catch unseen errors
```

Comment:
So why not remove the flag then?

Reply:
I didn't understand this. Images with tags need this flag but others don't.

Reply:
"shouldn't be added" sounds like you're adding it, but it shouldn't be.